code/modules/mob/living/carbon/xenomorph/castes/bull/castedatum_bull.dm

```diff
@@ -50,6 +50,12 @@
 	upgrade_name = "Young"
 
 	upgrade = XENO_UPGRADE_ZERO
+	actions = list(
+		/datum/action/xeno_action/xeno_resting,
+		/datum/action/xeno_action/regurgitate,
+		/datum/action/xeno_action/ready_charge/bull_charge,
+		/datum/action/xeno_action/activable/bull_charge,
+	)
```

Comment:
change the basetype datum not the young datum

Reply:
o my bad